packages/hash/frontend/src/pages/signup.page.tsx

```diff
@@ -1,15 +1,14 @@
 import React, { useEffect, useRef, useReducer } from "react";
-import { NextPage } from "next";
 import { useRouter } from "next/router";
 import { useMutation } from "@apollo/client";
 import { useUser } from "../components/hooks/useUser";
 
-import { SignupIntro as SignupIntroScreen } from "../components/auth/signup/SignupIntro";
-import { VerifyCode as VerifyCodeScreen } from "../components/auth/VerifyCode";
-import { AccountSetup as AccountSetupScreen } from "../components/auth/signup/AccountSetup";
-import { AccountUsage as AccountUsageScreen } from "../components/auth/signup/AccountUsage";
-import { OrgCreate as OrgCreateScreen } from "../components/auth/signup/OrgCreate";
-import { OrgInvite as OrgInviteScreen } from "../components/auth/signup/OrgInvite";
+import { SignupIntro } from "./signup.page/signup-intro";
+import { VerifyCode } from "./shared/verify-code";
+import { AccountSetup } from "./signup.page/account-setup";
+import { AccountUsage } from "./signup.page/account-usage";
+import { OrgCreate } from "./signup.page/org-create";
+import { OrgInvite } from "./signup.page/org-invite";
```

Comment:
I thought we'd end up with `import { OrgInviteScreen } from "./signup.page/org-invite-screen";` but not worth changing now I guess

Reply:
Will address in a follow up PR